src/main/java/com/jvm_bloggers/frontend/public_area/newsletter_issue/newsletter_panel/NewsletterIssuePanel.html

```diff
@@ -52,11 +52,6 @@ <h3 class="inset-1">Varia</h3>
             </div>
         </div>
 
-        <div style="padding: 0 0 1em 0">
-            <a href="https://2022.geecon.org/" target="_blank">
-                <img width="100%" alt="GeeCON 2022, Kraków 11-13 Maja" src="_custom/geecon_krk_2022.png"/>
```

Comment:
I think `/src/main/resources/static/_custom/geecon_krk_2022.png` can be removed as well

Reply:
Deleted as well 👍